src/gcp_scanner/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+def pytest_addoption(parser):
+  parser.addoption("--time", action="store_true")
+
+@pytest.fixture()
```

Comment:
I'd probably add a blurb that we should expand conf test to other tests too.

Reply:
We will likely have only one set of them for now but we can expand it to cover various other configs e.g. fetching creds from metadata vs service account key and etc.